yt_dlp/extractor/tvw.py

```diff
@@ -115,3 +116,53 @@ def _real_extract(self, url):
                 'is_live': ('eventStatus', {lambda x: x == 'live'}),
             }),
         }
+
+
+class TvwTvChannelsIE(InfoExtractor):
+    IE_NAME = 'Tvw:TvChannels'
+    _VALID_URL = r'https?://(?:www\.)?tvw\.org/tvchannels/(?P<id>[^/?#]+)'
+    _TESTS = [{
+        'url': 'https://tvw.org/tvchannels/air/',
+        'info_dict': {
+            'id': 'TVWAIR',
+            'ext': 'mp4',
+            'display_id': 'air',
+            'title': r're:^TVW Cable Channel Live Stream',
+            'thumbnail': r're:^https?://.*\.(?:jpe?g|png)$',
+            'live_status': 'is_live',
+        },
+    }, {
+        'url': 'https://tvw.org/tvchannels/tvw2/',
+        'info_dict': {
+            'id': 'TVW2',
+            'ext': 'mp4',
+            'display_id': 'tvw2',
+            'title': r're:^TVW-2 Broadcast Channel',
+            'thumbnail': r're:^https?://.*\.(?:jpe?g|png)$',
+            'live_status': 'is_live',
+        },
+    }]
+
+    def _real_extract(self, url):
+        display_id = self._match_id(url)
+        webpage = self._download_webpage(url, display_id)
```

Comment:
I think I would just leave the id from the URL as the `video_id`. `streamName` does not seem like a better id to me. It also means if the stream name were ever to not be present the stream could still be extracted.
```suggestion
        video_id = self._match_id(url)
        webpage = self._download_webpage(url, video_id)
```